gulpfile.js

```diff
@@ -326,12 +326,8 @@ gulp.task("installDevDeps", function(done) {
         "buble",
     ];
 
-    if (os.platform().match(/^win/)) {
-        let windowsCommands = ["start", "cmd.exe", "/K", "npm", "install", "--no-save", ...deps];
```

Comment:
why did you remove that one?

Reply:
I replaced it with `spawn('npm.cmd', ...)` on windows

This avoids a shell popping up, that has to be closed manually